src/Sulu/Bundle/AdminBundle/Resources/js/components/Form/grid.scss

```diff
@@ -9,6 +9,12 @@
 .grid-section {
     margin-bottom: -$fieldMarginBottom;
 
+    &:first-child {
+        .divider-container {
+            margin-top: -50px;
```

Comment:
I think I would not just use a magic number here, but instead move that into an own variable in the `Divider` component. Then this will continue to work if we change the top margin of the divider component.